app/controllers/lottery_controller.rb

```diff
@@ -78,21 +117,21 @@ def create_new_teams_for_bidding_response(teams, assignment, priority_info)
     end
   end
 
-  def merge_bids_from_different_previous_teams(assignment, team_id, user_ids, priority_info)
-    # Select data from `priority_info` variable that only related to team members in current team and transpose it.
-    # For example, below matrix shows 4 topics (key) and correponding priorities given by 3 team members (value).
+  def merge_bids_from_different_previous_teams(assignment, team_id, user_ids, user_bidding_info)
```

Comment:
I think we could change the assignment param to just sign_up_topics since we only use that and not other attributes of assignment in this method. Would save that first line of code.